tests/test_utils.py

```diff
@@ -624,6 +624,52 @@ def test_expand_file_path(tmpdir, index, root, expected):
     pd.testing.assert_index_equal(expanded_index, expected)
 
 
+@pytest.mark.parametrize(
+    'obj, expected',
+    [
+        (
+            audformat.segmented_index(),
+            False,
+        ),
+        (
+            audformat.segmented_index(
+                ['f1'] * 2,
+                [0, 1],
+                [2, 3],
+            ),
+            True,
+        ),
+        (
+            audformat.segmented_index(
+                ['f1', 'f2'],
+                [0, 1],
+                [2, 3],
+            ),
+            False,
+        ),
+        (
+            pd.Series(
+                index=audformat.segmented_index(
+                    ['f1'] * 2,
+                    [0, 2],
+                    [2, 3],
+                )
+            ),
+            False,
+        ),
+        (
+            pd.DataFrame(
+                index=audformat.filewise_index(['f1'] * 2)
```

Comment:
I would prefer to keep the`filewise` index test as before and add one for `DataFrame` in addition.